### PR TITLE
Audit FIXME's and TODO's

### PR DESCRIPTION
**Pull request by dylanmckay**
The codebase has had FIXME and TODO items dating back the last 5 years.

I have audited them, removing redundant ones and fixing others.

@agnat What do you think?

### CODE REVIEW

File: ' [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[AVR] Uncommented basic block printing statements
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+# On branch avr-support
+# Your branch is ahead of 'origin/avr-support' by 7 commits.
+#   (use "git push" to publish your local commits)
+#
+# Changes to be committed:
+#	modified:   lib/Target/AVR/AVRAsmPrinter.cpp
+#
```
Comment (agnat) [inline ']: Yo dawg! I commited the commit message... :-D

File is called "'" (single quote)...

Comment (dylanmckay) [inline ']: lol.. I'll fix that.

Comment (dylanmckay) [inline ']: Looks like I've already removed it in one of the other commits.

**Comment by agnat** [comment]
LGTM.

**Comment by dylanmckay** [comment]
Rebased and committed in 8787a6b2b5be408e3bf3e293676c44eb3297c3dd.

**Pull request closed without merging**